src/config.js

```diff
@@ -202,6 +203,7 @@ export default class Config {
       cafile: String(opts.cafile || this.getOption('cafile') || ''),
       cert: String(opts.cert || this.getOption('cert') || ''),
       key: String(opts.key || this.getOption('key') || ''),
+      concurrency: Number(opts.concurrency || this.getOption('concurrency') || constants.NETWORK_CONCURRENCY),
```

Comment:
we also have child concurrency, I think this option should be more specific to avoid confusion what kind of concurrency it is